cluster/ci/origin/pipelines/unit/Jenkinsfile

```diff
@@ -46,13 +71,28 @@ pipeline {
             "PULL_SECRET_NAME=${pullSecretRef}",
             "JOB_NAME=${env.JOB_NAME}",
             "BUILD_NUMBER=${env.BUILD_NUMBER}",
-            "REPO_OWNER=${env.REPO_OWNER ?: 'openshift'}",
-            "REPO_NAME=${env.REPO_NAME ?: 'origin'}"
+            "REPO_OWNER=${params.REPO_OWNER}",
+            "REPO_NAME=${params.REPO_NAME}",
+            "PULL_BASE_REF=${params.PULL_BASE_REF}",
+            "PULL_BASE_SHA=${params.PULL_BASE_SHA}",
+            "PULL_REFS=${params.PULL_REFS}",
+            "PULL_NUMBER=${params.PULL_NUMBER}",
+            "PULL_PULL_SHA=${params.PULL_PULL_SHA}"
           )
-
+          
           waitForPods(this, ["run": "${unitRunId}"], 7200)
         }
       }
     }
   }
+  post {
+    // Cleanup created pods
```

Comment:
Can we get `sinker` to do this?

/cc @kargakis

Reply:
Also need to think about cancellation here -- will pods get nuked when Jenkins job is killed?

Reply:
> Can we get sinker to do this?

That's my plan. Sinker today lists from a specific namespace and we use `ci` because we run the retester job there but we could either move the retester in `origin-ci` and switch `plank` and `sinker` to list from `origin-ci` or we could make sinker list from all namespaces.

Reply:
> Also need to think about cancellation here -- will pods get nuked when Jenkins job is killed?

Good question. I would expect them to stay around even on an aborted job.

Reply:
If included in the always block of the post section, it will always run. So if you cancel a test, it does go ahead and cleanup the pods created by the test. 
imho this seems less wasteful than cleaning up later. You don't need to let a ut run to completion if you're canceling the parent job.

There may be all kinds of reasons that pods don't get deleted though... maybe the jenkins slave fails to communicate with the master, the api call fails, etc. So it's not a bad idea to run sinker to keep things in shape, but I don't think that attempting to clean them up here is a bad thing.

Reply:
SGTM